index.html

```diff
@@ -23,7 +23,9 @@
 
 
     <div class="well container _container">
+        <div v-konami="displayDefrag"></div>
         <button v-on:click="show_arcs = !show_arcs" class="fab">Arcs</button>
+        <button v-if="show_defrag" v-on:click="defrag = !defrag" class="fab">Defrag</button>
```

Comment:
Now it just needs a `v-cloak` here, to avoid showing the button when the page is loading.

```suggestion
        <button v-if="show_defrag" v-on:click="defrag = !defrag" class="fab" v-cloak>Defrag</button>
```